src/RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -286,6 +286,8 @@ public JObject GetNep5Balances(JArray _params)
             foreach (var (key, value) in dbCache.Find(prefix))
             {
                 JObject balance = new JObject();
+                if (Blockchain.Singleton.View.Contracts.TryGet(key.AssetScriptHash) is null)
+                    continue;
```

Comment:
Maybe we shoud do that only in the first iteration, a break or a flag should be ok

Reply:
But we should list all assets in user's address, so we can't just break the iteration.